README.md

```diff
@@ -1,208 +1,262 @@
 
-Using the "Windows Phone Application Deployment" program you can deploy the XAP which PhoneGap creates onto your Windows device
+If everything is set up and you can build and deploy your application, you can proceed with the push notifications server.
+To establish connectivity between your notification server and the Apple Push Notification service you will need Apple Push Notification service SSL certificate in `.p12` format.
 
-<img src="assets/images/apploader.PNG"/>
+Follow these steps to obtain Apple Push Notification service SSL certificate from Apple:
 
-## Creating PhoneGap app
-In order to build a Mendix phonegap app that utilises the push notification application there are number of steps that you need to complete before being able to utilise the functionality.
+### Step 1 - Login to Apple Developer center
 
-### Step 1 - Login to Mendix home
-Open up home.mendix.com and navigate to the project that you wish to build the app for. Once on the project wall for your application click on the publish section.
+Login to Apple Developer and go to https://developer.apple.com/account/ios/certificate/create
 
-<img src="assets/images/step1.png"/>
+### Step 2 - Choose certificate's type
 
-### Step 2 - App Identifier
-Once in the publish section you must enter an app identifier for your application. This is important when setting up an IOS app. Refer to the section about creating an app identifier to find out more information about creating an app identifier and IOS certificate.
+Choose Push Notification service certificate. As you may see, there are two types of certificates: _development_ and _production_. Note that the development type certificate can only work with the sandbox environment. More about this explained later on in this guide.
 
-<img src="assets/images/step2.png"/>
+<img src="assets/images/apns/Cert-1.png"/>
 
-### Step 3 - Select Devices
-Select the devices that you want to deploy your app to and upload splash screen images for the devices you have selected. 
+### Step 3 - Select App ID
 
-Press the button publish to appstore and you will be asked whether you want to build in the cloud or do it yourself. Choose do it yourself and press the Download Phonegap Build Package.
+Pick your App ID from the dropdown list. If your app is not on the list, then you need to check your App ID entity. Most likely the push notification service is not turned on for the app.
 
+<img src="assets/images/apns/Cert-2.png"/>
 
-<img src="assets/images/step3.png"/>
+### Step 4 - CSR file
 
-### Step 4 - Download phonegap.zip
+On the next step you should be asked for your CSR file (Certificate Signing Request). You may use the same CSR that you used to create app signing certificate. If you don't have one, please follow the instruction within this step description.
```

Comment:
'*During* the next step'

'that you used to create *the* app signing certificate'

What is meant with  'please follow the instruction within this step'?